src/objc/abstract/MCOAbstractMessageRendererCallback.mm

```diff
@@ -11,17 +11,34 @@
 #import "MCOHTMLRendererDelegate.h"
 #import "MCOHTMLRendererIMAPDelegate.h"
 #import "MCOUtils.h"
+#import <Objc/runtime.h>
```

Comment:
This line is a big issue in the architecture. The C++ part is compatible with more than just Objective-C platforms.